### ISSUE TITLE
magnetic calibration

### ISSUE THREAD

**Issue by robotdreamers** [kriswiner/MPU6050#37]
Hi Kris, for 8-shape magnetic calibration. Do i need to move in 8-shape for all three dimensions (x, y, z)?
This what happens when I did on all directions on my chip. I am using uT units. And does it has to be around 0 like what you have done. Thank you very much.

![image](https://user-images.githubusercontent.com/34329345/46883674-ab5ea680-ce07-11e8-8ae1-76dd3361ccaa.png)

**Comment by kriswiner** [comment]
Yes all three directions, really over all possible directions over the
entire 3D response surface.

**Comment by robotdreamers** [comment]
I my location it looks like this, does it mean mZ = 480mG. what can I tell about mX and mY. Thanks.

![screen shot 2018-10-12 at 10 56 20 am](https://user-images.githubusercontent.com/34329345/46885674-85d49b80-ce0d-11e8-8f49-d6858855e994.png)

**Comment by kriswiner** [comment]
It means SQRT(Mx**2 + My**2 + Mz**2) ~ 480 mG, as expected. Look at the
USGS for local field in your area, probably not too different from
Danville, California.

**Comment by robotdreamers** [comment]
so how should I read this :). so many jargons for an amateur :(

![screen shot 2018-10-12 at 11 07 51 am](https://user-images.githubusercontent.com/34329345/46886284-5161df00-ce0f-11e8-946c-bbf39186a73b.png)

**Comment by kriswiner** [comment]
Huh?
227 mG, 54 mG, 420 mG is this what your mag shows?

**Comment by robotdreamers** [comment]
No i got from this website for my location: https://www.ngdc.noaa.gov/geomag-web/#igrfwmm

**Comment by robotdreamers** [comment]
ah so I am supposed I should expect: mX to be 227mG, mY 54mG and mZ 419mG.
